src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/DraftApprenticeship.cs

```diff
@@ -95,13 +95,16 @@ public void Merge(DraftApprenticeshipDetails source, Party modifyingParty)
                     break;
             }
 
-            if(source.DeliveryModel == Types.DeliveryModel.PortableFlexiJob)
+            if (source.DeliveryModel == Types.DeliveryModel.PortableFlexiJob)
             {
-                FlexibleEmployment = new FlexibleEmployment
-                {
-                    EmploymentPrice = source.EmploymentPrice,
-                    EmploymentEndDate = source.EmploymentEndDate,
-                };
+                FlexibleEmployment ??= new FlexibleEmployment();
+                FlexibleEmployment.EmploymentEndDate = source.EmploymentEndDate;
+                FlexibleEmployment.EmploymentPrice = source.EmploymentPrice;
+            }
+            else if (FlexibleEmployment != null)
+            {
+                FlexibleEmployment.EmploymentPrice = null;
+                FlexibleEmployment.EmploymentEndDate = null;
```

Comment:
Looks like we are going to be producing these records with null values

Reply:
Yes I discussed this with Ben, looks like theres no easy way to delete the record at the moment so its left as is